### PR TITLE
feat: Apps-Engine Deno Runtime update

### PR DESCRIPTION
**Pull request by d-gubert**
## Proposed changes (including videos or screenshots)

This PR refactors the usage of the Apps-Engine so that it is compatible with the new Deno Runtime, as implemented in https://github.com/RocketChat/Rocket.Chat.Apps-engine/pull/665
## Issue(s)

[PROJ-5](https://rocketchat.atlassian.net/browse/PROJ-5)
## Further comments

[PROJ-5]: https://rocketchat.atlassian.net/browse/PROJ-5?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: .github/workflows/ci-test-e2e.yml [hunk 1]
```diff
@@ -65,14 +65,15 @@ on:
...
+  DEBUG: appsEngine:*
   MONGO_URL: mongodb://localhost:27017/rocketchat?replicaSet=rs0&directConnection=true
   TOOL_NODE_FLAGS: ${{ vars.TOOL_NODE_FLAGS }}
   LOWERCASE_REPOSITORY: ${{ inputs.lowercase-repo }}
   DOCKER_TAG: ${{ inputs.gh-docker-tag }}
 
 jobs:
   test:
-    runs-on: ubuntu-20.04
+    runs-on: ae-deno-runtime-test
```
Comment (d-gubert) [inline .github/workflows/ci-test-e2e.yml:76]:
```suggestion
    runs-on: ubuntu-20.04
```

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -295,6 +295,7 @@ jobs:
   test-api:
     name: 🔨 Test API (CE)
     needs: [checks, build-gh-docker-coverage, release-versions]
+    runs-on: ae-deno-runtime-test
```
Comment (d-gubert) [inline .github/workflows/ci.yml:298]:
```suggestion
    runs-on: self-hosted
```

File: .github/workflows/ci-test-e2e.yml [hunk 3]
```diff
@@ -86,6 +87,13 @@ jobs:
     name: MongoDB ${{ matrix.mongodb-version }} (${{ matrix.shard }}/${{ inputs.total-shard }})
 
     steps:
+      - name: Collect Workflow Telemetry
+        uses: catchpoint/workflow-telemetry-action@v2
+        with:
+          theme: dark
+          job_summary: true
+          comment_on_pr: false
```
Comment (d-gubert) [inline .github/workflows/ci-test-e2e.yml:95]:
```suggestion
          comment_on_pr: false
          proc_trace_chart_max_count: 200
          metric_frequency: 1
```

File: apps/meteor/tests/end-to-end/apps/00-installation.js [hunk 4]
```diff
@@ -53,7 +53,7 @@ describe('Apps - Installation', function () {
 					})
 					.end(done);
 			});
-		});
+		}).timeout(20000);
```
Comment (d-gubert) [inline apps/meteor/tests/end-to-end/apps/00-installation.js:56]:
```suggestion
```

Comment (d-gubert) [inline apps/meteor/tests/end-to-end/apps/00-installation.js:56]:
```suggestion
		});
```

File: .github/workflows/ci-test-e2e.yml
Comment (tapiarafael) [inline .github/workflows/ci-test-e2e.yml:1]: Maybe we can avoid enabling debug logs by default. Let's enable them only when necessary to keep the logs cleaner.

File: apps/meteor/package.json
Comment (tapiarafael) [inline apps/meteor/package.json:1]: Are we going to use the tag `alpha`?

Comment (d-gubert) [reply apps/meteor/package.json:1]: That's just for the development cycle, before the release of the final candidate of RC the next version of the apps-engine is published to NPM and then we bump the candidate again

**Review by sampaiodiego — APPROVED** [review]

**Review by tapiarafael — APPROVED** [review]

**Pull request merged**